examples/truffle/package.json

```diff
@@ -23,7 +23,7 @@
   "homepage": "https://github.com/gnosis/ethcontract-rs",
   "devDependencies": {
     "@gnosis.pm/util-contracts": "^3.0.1",
-    "@openzeppelin/contracts": "^3.4.0",
+    "@openzeppelin/contracts": "4.0.0-beta.0",
```

Comment:
I needed to do this for the new solidity version. For some reason upgrading this (I think used a command like `yarn upgrade openzeppelin/contracts@4.0.0-beta.0`) has caused all the changes in the yarn.lock file. Not sure what is going on there.